custom_components/opnsense/switch.py

```
@@ -384,87 +454,65 @@
-
     @callback
     def _handle_coordinator_update(self) -> None:
         """Handle updated data from the coordinator."""
         dnsbl = self.coordinator.data.get(ATTR_UNBOUND_BLOCKLIST, {})
         if not isinstance(dnsbl, Mapping) or len(dnsbl) == 0:
-            self._attr_available = False
+            self._available = False
             return
-        self._attr_available = True
+        self._available = True
         self._attr_is_on = True if dnsbl.get("enabled", "0") == "1" else False
```

Ruff

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)